axelrod/fingerprint.py

```diff
@@ -0,0 +1,294 @@
+import axelrod as axl
+import numpy as np
+import matplotlib.pyplot as plt
+from axelrod.strategy_transformers import JossAnnTransformer, DualTransformer
+from axelrod.interaction_utils import compute_final_score_per_turn as cfspt
+from axelrod import on_windows
+from collections import namedtuple
+from tempfile import NamedTemporaryFile
+
+
+Point = namedtuple('Point', 'x y')
+
+
+def create_jossann(coordinate, probe):
```

Comment:
On reflection, I understand the merit of having these outside of the class, however some of these are quite particular to the Ashlock fingerprint and I can imagine that we will implement further fingerprints that don't make use of these functions **or** make use of modifications of them.

So I'm actually going to suggest that they (not all, will mark the ones I think this applies) get put back in the class (as static methods). They can still be 're-used' via inheritance if need be, and the test just requires a simple instance initialisation.

Before you make this change though @theref, I suggest you wait to hear what @meatballs and perhaps @marcharper have to say.